### ISSUE TITLE
[BUG]: mega man x7 and 8 Fmv subtitles keeps blinking

### ISSUE THREAD

**Issue by Yoshi20004** [PCSX2/pcsx2#5192]
### Describe the Bug

When in mega man x7 and x8 the fmv subtitles keep blinking in and out.

### Reproduction Steps

Thier is no fix to this bug

### Expected Behavior

The fmv subtitles do not bink in and out at all

### PCSX2 Revision

Pcsx2 1.7.0

### Operating System

Windows 11

### CPU

Ryzen 9 4900hs 

### GPU

Rtx 2060 max q

### GS Settings

Does not matter what settings are used still happens.

### Emulation Settings

Does not matter what settings are used still happens.

**Comment by Yoshi20004** [comment]
@refractionpcsx2 do know why this bug happened I know bug is minor bug talk probably try look fixed for it because  gitch been for a long time and really annoying gitch.

**Comment by refractionpcsx2** [comment]
is it fine in software mode?

**Comment by Yoshi20004** [comment]
No and say on pcsx2 wiki that thier is no known fix.

**Comment by refractionpcsx2** [comment]
Then I don't know the answer, we'll look in to it at some point.

**Comment by gilderoylockhart01** [comment]
In order for us to help you faster, can you make a blockdump of the game + a savestate when the glitch happens please?

**Comment by Yoshi20004** [comment]
It happens in cutscenes and know problem.

**Comment by Yoshi20004** [comment]
@refractionpcsx2 just add a gamedb fix for mega man x7 and x8 fmv blinking bug. After all Gamedb hack are magic bullet that fix any bug on pcsx2 without breaking any other  game.

**Comment by refractionpcsx2** [comment]
I'm not adding hacks for the sake of adding hacks, that's how we got a reputation of being hacky, we'd rather not go back there.

and It's not just the case of "just add a gamedb fix" when we don't know what the problem is.

**Comment by Yoshi20004** [comment]
@tellowkrinkle do know what problem is when fmv subtitles binking in and out kn mega man x7 and 8 because like how this bug might get fixed because should get fixed because once is they x8 moved to perfect.
Once idea is to add a gamedb hack after all gamedb hack are magic bullets that fixed this bug without breaking more games. If batter of problem they can find it and fix.

**Comment by refractionpcsx2** [comment]
Can you stop pinging random team members, please.

**Comment by Yoshi20004** [comment]
Just like know how long bug take be fixed because really should not hand to fixed because well known bug and last bug fixed for x7 and x8 can moved to perfect.

**Comment by tellowkrinkle** [comment]
Fun fact: GameDB hacks require us to know what the problem is first.  I can't guarantee we'll put one in, but I can guarantee that you *won't* get one if no one knows what the problem is.
Someone already told you what you should do if you want to help us find the problem.
> In order for us to help you faster, can you make a blockdump of the game + a savestate when the glitch happens please?

**Comment by Yoshi20004** [comment]
@tellowkrinkle so make save states right before intro cutscene.

**Comment by Yoshi20004** [comment]
And how do a blockdamp

**Comment by Yoshi20004** [comment]
@refractionpcsx2 how do blockdamp on pcsx2 emulator.

**Comment by refractionpcsx2** [comment]
> Just like know how long bug take be fixed because really should not hand to fixed because well known bug and last bug fixed for x7 and x8 can moved to perfect.

You're free to fix it, if it's not hard, PCSX2 is open source.

As for blockdumps, under the debug menu there is a "create blockdump" option, tick that, then run your game, then exit the emulator, there should be a .dump file in your pcsx2 folder.

**Comment by HeroponRikiBestest** [comment]
GSdump: [gs_20211220174457.gs.zip](https://github.com/PCSX2/pcsx2/files/7748330/gs_20211220174457.gs.zip)
Blockdump: [Mega Man X7 (USA) (2021-11-20 17-48-00).dump.zip](https://github.com/PCSX2/pcsx2/files/7748344/Mega.Man.X7.USA.2021-11-20.17-48-00.dump.zip)
Save State: [sstate.zip](https://github.com/PCSX2/pcsx2/files/7748337/sstate.zip)

**Comment by Yoshi20004** [comment]
Here inform for mega man x8 USA Ver
log file. [emuLog.txt](https://github.com/PCSX2/pcsx2/files/7748370/emuLog.txt)
- Blockdump.  [Mega Man X8 (2021-11-20 18-01-56).zip](https://github.com/PCSX2/pcsx2/files/7748387/Mega.Man.X8.2021-11-20.18-01-56.zip)
- save state. 
[sstates.zip](https://github.com/PCSX2/pcsx2/files/7748393/sstates.zip)

**Comment by Yoshi20004** [comment]
@tellowkrinkle @refractionpcsx2 OK guys get information can guys take look what problem is and how bug might get fixed because give staff need how guys need fixed the bug in question.

**Comment by Yoshi20004** [comment]
@refractionpcsx2 so look into logs and see problem is with fmv subtitles bug and tell us how going to fix it.

**Comment by refractionpcsx2** [comment]
If you don't stop spamming demanding we fix your bugs I'm just going to remove you from our github (again, I believe).  We'll get around to it when we get around to it, constantly bothering us isn't going to make us do it faster.

**Comment by Yoshi20004** [comment]
G

**Comment by gilderoylockhart01** [comment]
Btw the whole hack thing just makes me laugh at it. PCSX2 is the emulator that currently, almost have the less hacks of all the mature PS2 emulators out there. And the only hacks it have are often for the same thing (for exemple we have 2 hacks for vif fifo problems). Look the PS4 config file of Parapa the Rapper 2, you will see.

**Comment by Scalerize** [comment]
@refractionpcsx2 i'm using pcx2 v1.7.3582 and i can confirm that this issue no longer exists in mega man x8.

Here's a gs dump
https://anonfiles.com/xcMcp1J7y8/Mega_Man_X8_NTSC-U_SLUS-20960_20221125023714_gs

**Comment by RedDevilus** [comment]
For Megaman X8 I have added Blending TFF instead of Adaptive TFF for the subtitles. But it's just hiding the issue I suppose.

**Comment by refractionpcsx2** [comment]
yeah Blend is just hiding it, it's still present on both renderers, haven't looked in to why yet.

**Comment by refractionpcsx2** [comment]
Out of interest, I did a small amount of debugging and I came to the following.

Game works as expected on console.
Neither PCSX2 nor Dobiestation get it right
Playing back a GS Dump on the PS2 does not work either.

So the only conclusion I can draw from this is there is some sort of core issue, but I don't know what,  I tried various rounding/clamping settings etc, EE cache, instant DMA's vu's synced or unsynced, nothing helps.

So I'm kind of at a loss what the heck they're doing with these subtitles.

**Comment by Goatman13** [comment]
It's EE data cache issue. Is easy to miss on Megaman X7 because subtitles fade in, so at first it looks like they still blink, but not.
Both X7, and X8 work fine with EE cache emulation. You can even spot difference with blend tff with cache, subtitles are solid color then, not semitransparent.

**Comment by refractionpcsx2** [comment]
I thought it was cache, but it didn't fix it for me, I even booted the game with int + ee cache and it didn't work.

at least with x8

**Comment by Farran666** [comment]
I'm not an expert on this, but is it possible the fmv is running perfect, albeit at a slightly incorrect speed and a non-crt monitor?
